MockDuck/Sources/RequestResponseCommonProtocol.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  RequestResponseCommonProtocol.swift
+//  MockDuck
+//
+//  Created by Sebastian Celis on 6/18/19.
+//  Copyright © 2019 BuzzFeed, Inc. All rights reserved.
+//
+
+import Foundation
+
+protocol RequestResponseCommonProtocol {
+    var headers: [String: String]? { get }
+    var contentType: String? { get }
+}
+
+extension RequestResponseCommonProtocol {
+    var dataSuffix: String? {
+        guard let contentType = contentType else { return nil }
+
+        if contentType.contains("image/jpeg") {
+            return "jpg"
+        } else if contentType.contains("image/png") {
+            return "png"
+        } else if contentType.contains("image/gif") {
+            return "gif"
+        } else if contentType.contains("application/json") {
+            return "json"
```

Comment:
can we add 
```Swift 
else if contentType.contains("application/x-www-form-urlencoded") {
            return "txt"
        }
```
here?

Reply:
That sounds great to me but I would love for it to be done in its own pull request.

Reply:
I will open a separate PR for it after you merge this one

Reply:
Opened the PR here: https://github.com/buzzfeed/MockDuck/pull/27/files